infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/brokerphases/BrokerEnvironmentFactory.java

```diff
@@ -209,6 +209,8 @@ private Container newContainer(
     Container container = cb.build();
     Containers.addRamLimit(container, "250Mi");
     Containers.addRamRequest(container, "250Mi");
+    Containers.addCpuLimit(container, "300m");
+    Containers.addCpuRequest(container, "300m");
```

Comment:
@amisevsk wdyt about this defaults?

Reply:
Should be fine, though I haven't used the broker in a long while.